app/models/certificate.rb

```diff
@@ -130,6 +130,15 @@ def aia
       end
   end
 
+  def sia
+    @sia ||= get_extension('subjectInfoAccess')&.
+      split(/\n/)&.
+      map { |line| line.split(/\s*-\s*/, 2) }&.
+      each_with_object(Hash.new { |hash, key| hash[key] = [] }) do |(key, value), memo|
+        memo[key] << value
+      end
```

Comment:
WDYT of making this always return a hash, so that things that call this method don't need to do a nil check? I think all we'd need is one of these at the end:

```suggestion
      end || {}
```

Reply:
this was mostly copied from https://github.com/18F/identity-pki/blob/master/app/models/certificate.rb#L124-L131, I'll add this there too